### PR TITLE
Follow-up from Oct 7 discussion

### PR DESCRIPTION
**Pull request by stkent**
This probably won't build, just pushing up ideas i've been working on this evening for your reference.

### CODE REVIEW

File: app/src/main/java/com/github/stkent/amplify/DefaultQuestionView.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package com.github.stkent.amplify;
+
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+public class DefaultQuestionView implements QuestionView {
```
Comment (stkent) [inline app/src/main/java/com/github/stkent/amplify/DefaultQuestionView.java]: Decorate pattern, ish

File: app/src/main/java/com/github/stkent/amplify/Question.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
+package com.github.stkent.amplify;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+public class Question {
```
Comment (stkent) [inline app/src/main/java/com/github/stkent/amplify/Question.java]: Immutable representation of a question.

**Comment by stkent** [comment]
Merging w/ master now...

**Comment by stkent** [comment]
Investigating...

File: app/src/main/res/values/ids.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <item name="amp_positive_button" type="id" />
-    <item name="amp_negative_button" type="id" />
+    <item name="amplify_title_text_view" type="id" />
```
Comment (btkelly) [inline app/src/main/res/values/ids.xml]: should we stick to the amp prefix in all our resource files?

**Comment by stkent** [comment]
@btkelly :white_check_mark:

**Pull request merged**